timer/src/activities.ts

```diff
@@ -0,0 +1,25 @@
+import { config } from 'dotenv';
+import mailgun from 'mailgun-js';
+
+config();
```

Comment:
I would put this all in the worker where all initialization happens.
See my other pr for injecting the client into the activities

Reply:
Is the Worker where we want to formally recommend initializing config, etc.? That approach works for me, my only potential thought is that it might be a bit confusing because`worker.ts` currently doesn't explicitly access `activities.ts`, which may be a tad confusing.

Reply:
I’d say yes because the way the code is written today requires stubbing out a bunch of different things to make it testable